src/as2fm/jani_generator/jani_entries/jani_convince_expression_expansion.py

```diff
@@ -508,7 +508,7 @@ def expand_expression(
 
 
 def expand_distribution_expressions(
-    expression: JaniExpression, *, n_options: int = 101
+    expression: JaniExpression, *, n_options: int = 100
```

Comment:
```suggestion
    expression: JaniExpression, *, n_options: int
```

Reply:
For all the others, I would say we can remove the default value. For this one, I would keep the default value defined, since it is used in a number of tests and that value would be always the same.

Reply:
Nevermind, I removed it afterwards